polkadot/zombienet_tests/functional/0018-shared-core-idle-parachain.toml

```diff
@@ -36,4 +36,4 @@ chain = "glutton-westend-local-2000"
     name = "collator-2000"
     image = "{{CUMULUS_IMAGE}}"
     command = "polkadot-parachain"
-    args = ["-lparachain=debug"]
+    args = ["-lparachain=debug", "--experimental-use-slot-based"]
```

Comment:
I see this test still failed here https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/7763531. Could you please run it locally to make sure that it passes usually?

Reply:
Locally it passes with lookahead=4 but as we spoke offline it's concerning not to pass with lookahead=2 or even 3.